src/structured-data/structured-data-tags.js

```diff
@@ -158,16 +158,18 @@ function generateLiveBlogPostingData (structuredData = {}, story = {}, publisher
 function generateVideoArticleData (structuredData = {}, story = {}, publisherConfig = {}) {
   const metaKeywords = story.seo && story.seo['meta-keywords'] || [];
   const articleSection = get(story, ['sections', '0', 'display-name'], '');
+  const embedUrl = get(story, ['cards', '0', 'story-elements', '0', 'embed-url'], '');
```

Comment:
Don't even generate `embedUrl` instead of adding an empty if the first card has no video element.